addons/hr_org_chart/controllers/hr_org_chart.py

```diff
@@ -9,7 +9,7 @@
 class HrOrgChartController(http.Controller):
     _managers_level = 5  # FP request
 
-    def _check_employee(self, employee_id, **kw):
+    def _get_employee(self, employee_id, check_read_access=False, **kw):
         if not employee_id:  # to check
             return None
         employee_id = int(employee_id)
```

Comment:
I think it is not really useful, we can browse `False` for instance to have an empty recordset. 🙂

```suggestion
        employee_id = int(employee_id) if employee_id else False
```